HpToolsLauncher/Runners/AlmTestSetsRunner.cs

```diff
@@ -512,30 +530,79 @@ public TestSuiteRunResults RunTestSet(string tsFolderName, string tsName, double
                 Environment.Exit((int)Launcher.ExitCodeEnum.Failed);
             }
 
+            string xmlParameters = "";
+            if (!(tsParams.Equals("")) && !(tsParams == null))
```

Comment:
What does this if statement means? either refactor it to a method or put a comment